app/routes.js

```diff
@@ -1,11 +1,11 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
 const users = require('./controllers/users');
-const { passwordValidator } = require('./middlewares/users');
+const { passwordValidator, emailValidator } = require('./middlewares/users');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  app.post('/users', passwordValidator, users.signUp);
+  app.post('/users', passwordValidator, emailValidator, users.signUp);
```

Comment:
Generally, we join all middlewares in []. For example `[passwordValidator, emailValidator]`

Reply:
Also, I don't understand why do you check the email and password but you don't check the first_name and last_name 🤔